### PR TITLE
Add Job Application Email Address to Analytics ymls

### PR DESCRIPTION
**Pull request by josephhull676**
## Changes in this PR:

- Added the email address associated with job applications to both yml files so they're sent to big query

### CODE REVIEW

**Review by david-mears — APPROVED** [review]

File: config/analytics.yml [hunk 1]
```diff
@@ -118,6 +118,7 @@ shared:
     - in_progress_steps
     - reviewed_at
     - country
+    - email_address
```
Comment (alexbowen) [inline config/analytics.yml:121]: if i were being pedantic, i would say alphabetical order but you have been, its what was there before so dont worry...

**Review by alexbowen — APPROVED** [review]

File: config/analytics.yml (hunk 1 above)
Comment (josephhull676) [reply config/analytics.yml:121]: Yeah typically I would but I saw the rest weren't, so just stuck it on at the end!

Comment (alexbowen) [reply config/analytics.yml:121]: fair enough

Comment (david-mears) [reply config/analytics.yml:121]: Is it perhaps following the order in schema.db? If so that makes it easier to compare the two

**Pull request merged**